website/content/docs/enterprise/admin-partitions.mdx

```diff
@@ -11,7 +11,7 @@ description: >-
   This feature requires version 1.11.0+ of
   HashiCorp Cloud Platform (HCP) or self-managed Consul Enterprise.
   Refer to the{' '}
-  <a href="/docs/enterprise#consul-enterprise-feature-availability">enterprise feature matrix</a>
+  <a href="/consul/docs/enterprise#consul-enterprise-feature-availability">enterprise feature matrix</a>
```

Comment:
For some reason, the auto-translation to the new URLs (prefixing them with `/consul`) doesn't seem to apply to these links specified in manual `<a>` elements.